python/vulns_per_assets/page_asset_vulns.py

```diff
@@ -0,0 +1,135 @@
+import os
+import sys
+import time
+import math
+import requests
+
+# Might have to change this
+base_url = "https://api.kennasecurity.com/assets"
+ASSETS_PER_PAGE = 500
+
+# 
```

Comment:
These would be more prime candidates for a central library.

Reply:
I understand your comment.  It would take a re-organization of the code, and I'm attempting to get a blog out.